### PR TITLE
C spectral

### PR DESCRIPTION
**Pull request by vext01**
The last C benchmark. Much of the same thing.

OK?

### CODE REVIEW

File: benchmarks/spectralnorm/c/bench.c [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+ * Contributed by Sebastien Loisel
+ * Modified by Alex Belits
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+
+#define SPECTRAL_N 1000
+#define EXPEXT_CKSUM 1.2742241481294835914184204739285632967948913574218750
```
Comment (ltratt) [inline benchmarks/spectralnorm/c/bench.c]: Wasn't this called `EXPECT_CHKSUM` in the previous PR?

File: benchmarks/spectralnorm/c/bench.c [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+      for(j=0;j<N;j++)
+        {
+	  t0+=get_A(j,i)*u[j];
+	}
+      Au[i]=t0;
+    }
+}
+
+void eval_AtA_times_u(int N, const double u[], double AtAu[])
+{ double v[N]; eval_A_times_u(N,u,v); eval_At_times_u(N,v,AtAu); }
```
Comment (ltratt) [inline benchmarks/spectralnorm/c/bench.c]: Let's put these on separate lines -- it's really hard to see what's going on. [Even if the original had this, let's not perpetuate the ugliness.]

**Comment by vext01** [comment]
OK? Will squash if so.

**Comment by ltratt** [comment]
Squash away!

**Comment by vext01** [comment]
Crunch. Go ahead.

**Comment by vext01** [comment]
wait, something's wrong

**Comment by vext01** [comment]
No sorry, we are good ;)

**Pull request merged**